sources/Application/Views/PhraseView.cpp

```diff
@@ -198,6 +199,13 @@ void PhraseView::updateCursorValue(ViewUpdateDirection direction, int xOffset,
   if ((c) && (*c != 0xFF)) {
     int offset = offsets_[col_ + xOffset][direction];
 
+    // Add/remove from offset to match selected scale
+    int scale =
+      viewData_->project_->GetScale();
+    while (!scaleSteps[scale][(*c + offset) % 12]) {
+      offset>0?offset++:offset--;
```

Comment:
just personal preference but I find it a bit harder to read operators without spaces.